cli/templates/structure/index.md

```diff
@@ -20,14 +20,14 @@ The repository contains a manifest and directories with templates.
 
 ```
 📂 .keboola             - metadata directory
-┗ 🟦 repository.json    - manifest, templates paths and versions
-📂 [template]           - template directory, see structure below
+┗ 🟦 repository.json    - manifest, paths and versions
+📂 [template]           - template directory, see below
 ┗ 📂 [template version]
   ┗ ...
```

Comment:
Toto som musel skratit, ako je dlhsi obsah na pravej strane, tak to nevoslo na riadok.